test/install_script/nvm_bash_completion

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+cleanup () {
+  unset -f die nvm_detect_profile
+  rm -f test_profile > /dev/null 2>&1
+}
+
+die () { echo $@ ; exit 1; }
+
+NVM_ENV=testing . ../../install.sh
+
+nvm_detect_profile () {
+  echo "$HOME/test_profile"
```

Comment:
I didn't get your point here, remove it won't make you cannot write to it again.